bin/create_hugo_data.py

```diff
@@ -494,13 +494,149 @@ def export_events(anthology, builddir, dryrun):
         else:
             data["title"] = f"{anthology.venues[main_venue].name} ({data['year']})"
 
+        # Add talks data if available
+        if event.talks:
+            talks_data = []
+            for idx, talk in enumerate(event.talks, 1):
+                # Generate talk ID from video filename if available, otherwise use default pattern
+                if "video" in talk.attachments and talk.attachments["video"].name:
+                    # Extract talk ID from video filename like "2022.acl-keynote.2.mp4"
+                    video_name = talk.attachments["video"].name
+                    if video_name.endswith(".mp4"):
+                        # Remove .mp4 extension to get the talk ID
+                        talk_id = video_name[:-4]
+                    else:
+                        talk_id = video_name
+                else:
+                    # Fallback to sequential numbering if no video
+                    talk_id = f"{event.id}.talk-{idx}"
```

Comment:
Hmm, I'm not sure I like I creating these IDs within the build script. It means that it's not possible to find the talk via this ID through our Python library.

Another issue is that the talk ID will change if we change the filename for some reason, which is not how other IDs work.

One idea would be to make the ID explicit in the XML and expose it through the Python library.